GoogleForCreators/web-stories-wp
# Pre-Publish Checklist: Bring back warning for Update button

## Summary


Both Publish and Update buttons should trigger and show tooltip and warning icon if there are prepublish checklist errors.


![image-1](https://user-images.githubusercontent.com/1738349/100890897-587c5900-347e-11eb-81d3-3622f001d276.png)
![image](https://user-images.githubusercontent.com/1738349/100890898-5914ef80-347e-11eb-94cc-b51dcdff141c.png)

## Relevant Technical Choices


This seemed to have gone away because of a feature flag being deprecated. I consolidated the logic between the update and publish icons in this PR to make this logic even more centralized.

## To-do



## User-facing changes



## Testing Instructions


Test with both a draft and published story:

1. Create/open story with prepublish checklist errors
2. Hover over publish/update button and both a tooltip ("There are items in the checklist to resolve") and icon should show
3. Fix errors and re-hover over button. Icon and tooltip should go away.

---



Fixes #5972
